### DISCUSSION TITLE
Kwikset lock always report an empty status for Door Lock Logging record

### DISCUSSION THREAD

**Discussion by spudwebb** [zwave-js/zwave-js#7171] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

When trying to get the latest Door Lock Logging record from a Kwikset Lock 910 the report has the status field wrongly set to empty (=0) so that zwavejs does not parse any other information from the message sent.

query:
```
2024-09-12T14:33:36.708Z SERIAL » 0x012500a90001004c1798815875f0601204d4d10717b5db45b35e25797d2dcdbf2 (39 bytes)
                                  500000000091a
2024-09-12T14:33:36.709Z DRIVER » [Node 076] [REQ] [SendDataBridge]
                                  │ source node id:   1
                                  │ transmit options: 0x25
                                  │ callback id:      9
                                  └─[SecurityCCCommandEncapsulation]
                                    │ nonce id: 69
                                    └─[DoorLockLoggingCCRecordGet]
                                        record number: 0
```

report:
```
2024-09-12T14:33:37.013Z SERIAL « 0x012f00a8000001004c219881151898eb7af12510d9c94e6a788d10b307a460d04 (49 bytes)
                                  30cbeb3b5d3c75c3b0aa300b2007f7f23
2024-09-12T14:33:37.014Z SERIAL » [ACK]                                                                   (0x06)
2024-09-12T14:33:37.017Z DRIVER « [Node 076] [REQ] [BridgeApplicationCommand]
                                  │ RSSI: -78 dBm
                                  └─[SecurityCCCommandEncapsulation]
                                    │ sequenced: false
                                    └─[DoorLockLoggingCCRecordReport]
                                        record #: 6 (empty)
```
I manually parsed the message and it contains valid data:
```
eventType:8
label:'Manually Locked'
timestamp:'2024-09-11T16:14:28.000Z'
userCode:''
userId:0
```

Would it be possible to always try to parse all the properties from a Door Lock Logging record, and let the calling application decide what to do when the status property is empty. 
Or alternatively add a compat flag to ignore the status property for specific locks?
 

### Software versions

Driver (node-zwave-js): 13.1.0

### Device information

Manufacturer: Kwikset
Model name: 910
Node ID: 76

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_2024-09-12.log](https://github.com/user-attachments/files/17018079/zwavejs_2024-09-12.log)

**Comment by AlCalzone** [comment]
I've pinged one of their developers, so maybe this can be fixed in firmware.
As for current devices, I prefer a compat flag to try and parse the record anyways.
